Update serverless installation docs (#8116)
DataDog/documentation: content/en/serverless/installation/python.md

      text: 'Installing Ruby Serverless Monitoring'
---

> After you have [installed the AWS integration][1], choose one of the following method to instrument your application to send metrics, logs, and traces to Datadog.

Reviewer: ```suggestion
After you have [installed the AWS integration][1], choose one of the following methods to instrument your application to send metrics, logs, and traces to Datadog.
```

## Configuration

{{< tabs >}}
{{% tab "Serverless Framework" %}}

Use the [Datadog Serverless Plugin][1] to send metrics, logs and traces from your application to Datadog without any code changes. The plugin automatically attaches the Datadog Lambda Library for Python to your functions using layers. At deploy time, it swaps your original function handler with the Datadog-provided handler that initializes the Datadog Lambda Library and invokes your original function handler. It also enables Datadog and X-Ray tracing for your Lambda functions.

To install and configure the Datadog Serverless Plugin, follow these steps:

1. Install the Datadog Serverless Plugin: 
	  ```
    yarn add --dev serverless-plugin-datadog
    ```
2. In your `serverless.yml`, add the following:
    ```
    plugins:
      - serverless-plugin-datadog
    ```
3. In your `serverless.yml`, also add the following section:
    ```
    custom:
      datadog:
        flushMetricsToLogs: true
        forwarder: # The Datadog Forwarder ARN goes here.
    ```

>     For more information on the Forwarder ARN, or to install the forwarder see the [forwarder documentation][2].

Reviewer 2: ```suggestion
    For more information on the Forwarder ARN, or to install the forwarder see the [Forwarder documentation][2].
```

[1]: https://console.aws.amazon.com/cloudformation/home#/stacks?filteringText=forwarder
{{% /tab %}} --->
{{% tab "Custom" %}}

### Install the Datadog Lambda Library

You can either install the Datadog Lambda library as a layer or as a Python package.

The minor version of the `datadog-lambda` package always matches the layer version. E.g., datadog-lambda v0.5.0 matches the content of layer version 5.

#### Using the Layer

> [Configure the layers][1] for your Lambda function using the ARN in the following format.

Reviewer: ```suggestion
[Configure the layers][1] for your Lambda function using the ARN in the following format:
```

arn:aws:lambda:us-east-1:464622532012:layer:Datadog-Python37:11
```

The available `RUNTIME` options are `Python27`, `Python36`, `Python37`, and `Python38`. For `VERSION`, see the [latest release][2].

#### Using the Package

```
pip install datadog-lambda
```

Or add `datadog-lambda` to your project's `requirements.txt`. See the [latest release][3].

### Configure the Function

1. Set your function's handler to `datadog_lambda.handler.handler`.

> 2. Set the environment variable `DD_LAMBDA_HANDLER` to your original handler, e.g., `myfunc.handler`.

Reviewer: ```suggestion
2. Set the environment variable `DD_LAMBDA_HANDLER` to your original handler, for example, `myfunc.handler`.
```

> 3. Set environment variable `DD_TRACE_ENABLED` to `true`.

Reviewer: ```suggestion
3. Set the environment variable `DD_TRACE_ENABLED` to `true`.
```

> 4. Set environment variable `DD_FLUSH_TO_LOG` to `true`.

Reviewer: ```suggestion
4. Set the environment variable `DD_FLUSH_TO_LOG` to `true`.
```

5. Optionally add a `service` and `env` tag with appropriate values to your function.

### Subscribe the Datadog Forwarder to the Log Groups

> You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.

Reviewer: ```suggestion
You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, to send metrics, traces and logs to Datadog.
```

> 1. [Install the Datadog Forwarder if you haven't][4].

Reviewer: ```suggestion
1. [Install the Datadog Forwarder][4] if you haven't.
```

> 2. [Ensure the parameter DdFetchLambdaTags is set to `true`][5].

Reviewer: ```suggestion
2. Set the parameter [`DdFetchLambdaTags`][5] to `true`.
```

3. [Subscribe the Datadog Forwarder to your function's log groups][6].

[1]: https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html
[2]: https://github.com/DataDog/datadog-lambda-layer-python/releases
[3]: https://pypi.org/project/datadog-lambda/
[4]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder
[5]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder/#ddfetchlambdatags
[6]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#send-aws-service-logs-to-datadog
{{% /tab %}} 
{{< /tabs >}}

> ## Quick Start

Reviewer 2: ```suggestion
## Explore Datadog Serverless Monitoring
```

> ## Quick Start

Reviewer: ```suggestion
## Viewing your data
```

> After you have configured your function following the steps above, you can view the enhanced metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below.

Reviewer: ```suggestion
After you have configured your function following the steps above, you can view the enhanced metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below:
```

Reviewer 2: fixed

> After you have configured your function following the steps above, you can view metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below.

Reviewer 2: ```suggestion
After you have configured your function following the steps above, you can view metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below:
```
